config/database.yml

```diff
@@ -1,25 +1,25 @@
-# SQLite version 3.x
-#   gem install sqlite3
+# MySQL version 2.x
+#   gem install mysql2
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
+#   Ensure the MySQL 2 gem is defined in your Gemfile
+#   gem 'mysql2'
 #
+
 default: &default
-  adapter: sqlite3
+  adapter: mysql2
   pool: 5
-  timeout: 5000
+  username: <%= ENV['DATABASE_USER'] || "dev" %>
```

Comment:
For this to match the configuration for the database used for web that we use locally, can this be switched to `rr-dev` for the default?